BUILD.gn

```diff
@@ -646,6 +646,13 @@ action("create_dist") {
     ]
   }
 
+  if (is_linux) {
+    deps += [ 
+      "//electron/app/linux:strip_and_dump_brave_symbols",
```

Comment:
this works, but we could keep these inside the linux BUILD.gn if you made them dependencies of `//electron/app/linux:dist_resources`